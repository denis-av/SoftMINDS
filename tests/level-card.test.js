import React from 'react';
import LevelCard from '../components/level-card';

import renderer from 'react-test-renderer';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

test('match correct the snapshot', () => {
  const tree = renderer.create(<LevelCard />).toJSON();
  expect(tree).toMatchSnapshot();
});