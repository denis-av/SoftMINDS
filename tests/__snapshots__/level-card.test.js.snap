// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match correct the snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": undefined,
      "borderRadius": 20,
      "height": 90,
      "justifyContent": "center",
      "width": 90,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontFamily": "bold-font",
        "fontSize": 35,
      }
    }
  />
</View>
`;
